indico/modules/auth/blueprint.py

```diff
@@ -34,3 +36,14 @@
 _bp.add_url_rule('/register/<provider>', 'register', RHRegister, methods=('GET', 'POST'))
 
 _bp.add_url_rule('/reset-password/', 'resetpass', RHResetPassword, methods=('GET', 'POST'))
+
+
+with _bp.add_prefixed_rules('/user/<int:user_id>', '/user'):
+    _bp.add_url_rule('/accounts/', 'accounts', RHUserAccounts, methods=('GET', 'POST'))
+    _bp.add_url_rule('/accounts/remove/<identity>', 'remove-account', RHUserAccountsRemove)
+
+
+@_bp.url_defaults
+def _add_user_id(endpoint, values):
+    if (endpoint == 'auth.user_profile' or endpoint.startswith('api.key_')) and 'user_id' not in values:
```

Comment:
Copy&paste error.

    if endpoint in {'auth.accounts', 'auth.remove-account'} and 'user_id' not in values:

(should of course be adapted to whatever the final name of those endpoints will be)